source/crud/crud.rst

```diff
@@ -1328,11 +1471,10 @@ Q & A
 =====
 
 Q: Why do the names of the fields differ from those defined on docs.mongodb.org?
-  Documentation and commands often refer to same-purposed fields with different names making it difficult to have a cohesive API. In addition, occasionally the name was correct at one point and its purpose has expanded to a point where the initial name doesn't accurately describe its current function.
+  documentation and commands often refer to same-purposed fields with different names making it difficult to have a cohesive API. In addition, occasionally the name was correct at one point and its purpose has expanded to a point where the initial name doesn't accurately describe its current function.
```

Comment:
This should still be capitalized, as it's the start of a sentence.

Also, the line above refers to "docs.mongodb.org". You may want to change that to "the MongoDB manual".